pkgs/applications/graphics/yed/default.nix

```diff
@@ -35,5 +35,6 @@ stdenv.mkDerivation rec {
     description = "A powerful desktop application that can be used to quickly and effectively generate high-quality diagrams";
     platforms = jre.meta.platforms;
     maintainers = with maintainers; [ abbradar ];
+    broken = !("gtk3" ? jre);
```

Comment:
Not sure on the expression but that error is annoying on darwin.

Reply:
Ummmmmmm. This breaks on all systems. It must be `jre ? gtk3` and not the other way around!